app/dashboard/templates/yge/send1.html

```diff
@@ -21,7 +21,7 @@
         </span>
 		<h1>Send Tip ⚡</h1>
 		<h3>It's Fast.  It's Easy.  It's Free.️</h3>
-        <h4>(Supports any github username or email address)</h4>
+        <p>(Supports any github username or email address)</p

```

Comment:
oh nooo... I commited broken HTML. 
As this PR is already merged, this is fixed in: https://github.com/gitcoinco/web/pull/202/commits/a6a4c7d8ec8e9c10c60da3edc4aff2ec38e8c5e1#diff-88a1050efecc8245bddec63cb35b9412R24

BUT as that other PR is kinda big, shall I do a quick PR to fix only this error, that we easily can merge?